Example Apps/Example Swift/RemoteControlManager.swift

```diff
@@ -0,0 +1,237 @@
+//
+//  RemoteControlManager.swift
+//  SmartDeviceLink-Example-Swift
+//
+//  Created by Beharry, Justin (J.S.) on 7/28/22.
+//  Copyright © 2022 smartdevicelink. All rights reserved.
+//
+
+import Foundation
+import SmartDeviceLink
+import SmartDeviceLinkSwift
+
+class RemoteControlManager {
+    private let sdlManager: SDLManager
+    private let homeButtons: [SDLSoftButtonObject]
+    private var remoteControlCapabilities: SDLRemoteControlCapabilities?
+    private var climateModuleId: String?
+    private var hasConsent: Bool?
+    private var climateData: SDLClimateControlData?
+
+    public let isPermissionEnabled: Bool
+    public var climateDataString: String {
```

Comment:
```suggestion
    let isEnabled: Bool
    var climateDataString: String {
```

Because this is not a framework, `public` is effectively equivalent to `internal` (which is the same as no visibility keyword). If it were a framework, then it would be important.